MonMaraicher/Features/Map/MapViewModel.swift

```diff
@@ -42,6 +44,28 @@ final class MapViewModel: ObservableObject {
         CLLocationManager().requestWhenInUseAuthorization()
         CLLocationManager().desiredAccuracy = kCLLocationAccuracyBest
     }
+
+    func onButtonTapped() {
+        if let nearbyFarmer = findNearbyFarmer() {
+            mapCameraPosition = .region(MKCoordinateRegion(center: CLLocationCoordinate2D(latitude: nearbyFarmer.location.latitude, longitude: nearbyFarmer.location.longitude), span: MKCoordinateSpan(latitudeDelta: 0.01, longitudeDelta: 0.01)))
+        } else {
+            // TODO: request to the user to invite them to accept the location
+        }
+    }
+
+    private func findNearbyFarmer() -> Farmer? {
+        var nearbyDistance = Double.infinity
```

Comment:
Infinity won't give you the max value. Infinity is not a number in maths.

Is your map working with that value? Did you try it? I bet I won't work as expected ;)

You could choose a rational value as maximum range.

Reply:
Or handle that case differently. Understand why you can be in that case

Reply:
Yes, I tried on the simulator in different places on the map, the result obtained is good.
But I agree with you, using Double.infinity seems to me not optimized